### ISSUE TITLE
Special characters problem

### ISSUE THREAD

**Issue by d3sign3r** [sergiodlopes/jquery-flexdatalist#78]
Ok. Back again with a new issue that i cant figure it out this time :(
My problem is that i use special characters in my json file that can't be recognized by the plug-in. What i mean is that in Greek Language the letters `α, ά` or `ε, έ` are the same letters but if you type the letter `α` in the search input field it doesn't highlight also the letters `ά` in the results list ! How can i fix this issue Sergio?

Thanks!

**Comment by sergiodlopes** [comment]
Hi,

Are you knowledgeable with javascript?
Because of this kind of problems I made available a callback option so that you can manipulate the string/word comparison.
 
Check out option `normalizeString`:
http://projects.sergiodinislopes.pt/flexdatalist/

Code (`normalizeSpecialCharacters` function must be implemented by you, that can replace special characters):
$.flexdatalist({
     [..]
     normalizeString: function (string) {
          return normalizeSpecialCharacters(string);
     }
});

In the documentation, you'll find a reference to a plugin:
https://github.com/dundalek/latinize

Would this plugin help solve your problem?

Sérgio

**Comment by d3sign3r** [comment]
Thanks for quick reply Sergio. i m not so sure about "latinize" plug-in but i think that this plug-in just replace one character to another eg. "ά" to "α" etc. 
What i m trying to do is to make flexdatalist understand that "ά" is equal to "α" and vice versa, NOT REPLACE "ά" with "α" etc. 
Is this possible with "Latinize" plug-in ? 

Once again thanks for looking into my issue.

**Comment by sergiodlopes** [comment]
Fair point. Yeah, the plugin will replace "ά" with "α" just for comparison sake in the search (not changing the source string).

Unfortunately, AFAIK, javascript regex is not very good with special characters. :\

One solution, that I'm trying to avoid, is having all special characters mapped to non-special characters... but, that would considerably make the plugin bigger in size.

If someone more knowledgeable can help with this, I would appreciate it.

Sérgio

**Comment by d3sign3r** [comment]
Hi Sergio. i tried almost everything but doesn't work :( 
My best approach was this : 

i wrote a js code that convert "ά" to "α" but this happens inside the input field and not inside the hilight function of your plugin. 
```
<script type="text/javascript">
$(function() {
    $(".location-search-field").keyup(function() {

        $(this).val($(this).val().replace(/ά/g, "α"));
        
        });
});
        </script>
```
is there any way to implement that code to take action inside the hilight function?

**Comment by sergiodlopes** [comment]
Hi,

If you can make the plugin work for you case, great. It's definitely a problem that I'll keep thinking on a solution.
Working in portuguese language projects, I face the same problem/limitation. :\

Sérgio

**Comment by marine1988** [comment]
hi there sergio is there any solution for this?

**Comment by gitNovaCorp** [comment]
hi there sergio is there any solution for this?
